lib/app_jwt/app_jwt.c

```diff
@@ -14,7 +20,12 @@
 #include <date_time.h>
 #include <psa/crypto.h>
 #include <psa/crypto_extra.h>
+
+#if defined(CONFIG_BOARD_NATIVE_SIM)
+#define IAK_APPLICATION_GEN1 0x41020100
+#else
 #include <psa/nrf_platform_key_ids.h>
```

Comment:
Why can't this be included with native sim?

Reply:
native_sim doesn't use the same PSA implementation. The native one doesn't have that header file.